src/_includes/compatibility-vbe.md

```diff
@@ -0,0 +1,92 @@
+{% assign packages = site.data.composer_lock.packages %}
+
+{% assign versionsNumber = include.data.versions.size %}
+{% assign extensions = include.data.extension %}
+{% if include.extension %}
+{% assign extensions = include.data.extension | where: "name", include.extension %}
+{% endif %}
+
+**Supported**{: .status-light.supported } – version that has been fully tested by Magento and is supported.
+
+<!-- **Compatible**{: .status-light.compatible } – independent release version that has not been fully tested by Magento, but is confirmed to be compatible. -->
+
+**Not&nbsp;supported**{: .status-light.not-supported } - version that is not compatible with a Magento release.
```

Comment:
Is it essential to have this non-breaking space thingy?

Reply:
I don't think so... I'm not sure what the original purpose was. I removed it.